### PR TITLE
DEV-9203 atd popout

### PR DESCRIPTION
**Pull request by nick-torres**
**High level description:**

Copied the minimum amount of components I could from the glossary in order to create the new drawer for about the data

**JIRA Ticket:**
[DEV-9203](https://federal-spending-transparency.atlassian.net/browse/DEV-9203)

**Mockup:**
https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55/screen/633ca36925860428cbcf2662

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/glossary/glossaryPage.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .usa-da-glossary-animations {
-    // @import "all"; this was absolutely killing me.
```
Comment (brianpetway) [inline src/_scss/pages/glossary/glossaryPage.scss:2]: Can probably just delete this line.

File: src/js/components/aboutTheDataUpdated/AboutTheData.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
```
Comment (brianpetway) [inline src/js/components/aboutTheDataUpdated/AboutTheData.jsx:1]: If this is a new file you should add the 'Created By' block at the top.

File: src/js/components/aboutTheDataUpdated/AboutTheDataHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useRef } from 'react';
```
Comment (brianpetway) [inline src/js/components/aboutTheDataUpdated/AboutTheDataHeader.jsx:1]: If this is a new file you should add the 'Created By' block at the top.

**Comment by brianpetway** [comment]
A few inline comments.

Also, the animation doesn't seem to be working. The drawer just appears without sliding out, like the glossary does.

And the ticket has very little info in it. More description about exactly what's expected is needed. And also some breadcrumbs, nowhere in the ticket does it say you need to go to the Spending Explorer to see this new feature.

**Comment by nick-torres** [comment]
added a bunch more content to the ticket to make it easier to actually test

**Review by brianpetway — APPROVED** [review]

**Pull request merged**